libnetwork/drivers/bridge/bridge_store.go

```diff
@@ -146,7 +146,7 @@ func (ncfg *networkConfiguration) MarshalJSON() ([]byte, error) {
 	nMap["Internal"] = ncfg.Internal
 	nMap["DefaultBridge"] = ncfg.DefaultBridge
 	nMap["DefaultBindingIP"] = ncfg.DefaultBindingIP.String()
-	nMap["HostIP"] = ncfg.HostIP.String()
+	nMap["HostIPv4"] = ncfg.HostIPv4.String()
```

Comment:
That's an incompatible change to the on-disk format. Making it a compatible change is thankfully trivial:
```suggestion
	nMap["HostIP"] = ncfg.HostIPv4.String()
```

Reply:
Done.